app/src/userpages/components/StreamPage/index.jsx

```diff
@@ -45,22 +44,22 @@ const StreamPage = (props) => {
 
     const stream = useSelector(selectOpenStream)
 
-    const editedStream = useSelector(selectEditedStream)
-
     const currentUser = useSelector(selectUserData)
 
     const isMounted = useIsMounted()
 
-    const { id } = stream || {}
+    const { isNewStream } = useNewStreamMode()
```

Comment:
Why `{ isNewStream } =` and not `isNewStream =`?

Reply:
No other reason besides consistency with `useNewProductMode` hook. Now I noticed there is also `useEmbedMode` that only returns boolean. I'll change this and the product hook to return only boolean.